app/support/notification_sender.rb

```diff
@@ -65,29 +62,33 @@ def order_details
                                   .includes(:product, :order, :price_policy, :reservation)
   end
 
-  def account
-    Account.find(@accounts_to_notify.map(&:first)).map(&:account_list_item)
-  end
-
   def reviewed_at
     @reviewed_at ||= Time.zone.now + Settings.billing.review_period
   end
 
   class AccountNotifier
 
-    def notify_accounts(accounts_to_notify)
-      accounts_to_notify.each do |account_id, facility_id|
-        account = Account.find(account_id)
-        account.notify_users.each do |u|
-          Notifier.review_orders(user_id: u.id, facility_id: facility_id, account_id: account_id).deliver_now
+    def notify_accounts(account_ids_to_notify)
+      notifications_hash(account_ids_to_notify).each do |user_id, account_ids|
+        Notifier.review_orders(user_id: user_id, account_ids: account_ids).deliver_now
+      end
+    end
+
+    private
+
+    def notifications_hash(account_ids_to_notify)
```

Comment:
This method name is a little vague. Not sure what better to call it, though. Maybe add a comment about the structure it's expected to return?